Makefile.am

```
@@ -169,3 +169,4 @@ DISTCHECK_CONFIGURE_FLAGS = --enable-man
 clean-local:
 	rm -rf test_bitcoin.coverage/ zcash-gtest.coverage/ total.coverage/
 	rm -rf afl-temp
+	rm -rf zcutil/bin
```

Hmm, actually I think I agree with @daira now that this should be cleaned by `make distclean`, as it's built by the depends system. Let's make this match `zcutil/distclean.sh`:
```suggestion

distclean-local:
	rm -f zcutil/bin/db_*
	rmdir zcutil/bin 2>/dev/null || true
```